website_and_docs/content/documentation/webdriver/drivers/_index.en.md

```diff
@@ -61,21 +61,25 @@ and it is recommended to always use `quit` to end the session
 
 {{< tabpane text=true >}}
 {{< tab header="Java" >}}
+{< gh-codeblock path="examples/java/src/test/java/dev/selenium/getting_started/FirstScript.java#L29" >}
 {{< badge-code >}}
 {{< /tab >}}
 {{% tab header="Python" %}}
 {{< gh-codeblock path="examples/python/tests/drivers/test_options.py#L13" >}}
 {{% /tab %}}
 {{< tab header="CSharp" >}}
+{< gh-codeblock path="examples/java/src/test/java/dev/selenium/getting_started/FirstScript.java#L29" >}
```

Comment:
Hi @shbenzer,

I noticed that the changes are already in the PR #1882. Could you please do the following:

Update the existing PR with the necessary changes (or) you can update this PR with the comments provided in [PR #1882](https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1882) and then you can close PR #1882.

Thanks,
Sri

Reply:
Yep! I'll do that and close pr #1882

Reply:
Done